index.html

```diff
@@ -2464,32 +2471,40 @@ <h3>Data Schemas</h3>
     "https://www.w3.org/ns/credentials/examples/v2"
   ],
   "id": "http://university.example/credentials/3732",
-  "type": ["VerifiableCredential", "ExampleDegreeCredential"],
+  "type": ["VerifiableCredential", "ExampleDegreeCredential", "ExampleNameCredential"],
   "issuer": "https://university.example/issuers/14",
   "validFrom": "2010-01-01T19:23:24Z",
   "credentialSubject": {
     "id": "did:example:ebfeb1f712ebc6f1c276e12ec21",
     "degree": {
       "type": "ExampleBachelorDegree",
       "name": "Bachelor of Science and Arts"
+    },
+    "name": {
+      "firstName": "John",
+      "lastName": "Doe"
```

Comment:
```suggestion
    "name": {
      "firstName": "John",
      "lastName": "Doe"
```

Example above doesn't follow best practices: https://schema.org/name

See the example here (JSON-LD tab, for now `name` is typically used): https://schema.org/Person#eg-0001

... when in doubt, look at schema.org for examples. You could do `alumniOf` (or `memberOf`):

https://schema.org/alumniOf

```
  "alumniOf": {
    "name": "Example University"
   }
```

Reply:
thanks, updated to `alumniOf`